.env.test.example

```diff
@@ -0,0 +1,5 @@
+# Test database credentials
+DB_NAME=starttls_test
```

Comment:
Should this be `TEST_DB_NAME`?

Reply:
As written, `TEST_DB_NAME` is only used to look up the default for `DB_NAME` while in test mode. But, I could swap for TEST_DB_NAME throughout in test mode if you think that's clearer.

Reply:
Oh, I see! Yeah, let's make the usage of `TEST_DB_NAME` explicit in `LoadEnvironmentVariables`-- so if `DB_NAME` is set, we still don't accidentally nuke our actual DB!